site/docs/reference/connectors.md

```diff
@@ -71,68 +71,69 @@
 
 
 
-<a name="policy_manager_request.proto"></a>
+<a name="policy_manager_response.proto"></a>
```

Comment:
@roee88 Haven't the protos been removed as part of the taxonomy work?

Reply:
Not yet (we support both until we finish all work)